src/Shellbar/Shellbar.js

```diff
@@ -490,6 +497,18 @@ Shellbar.propTypes = {
     logoSAP: PropTypes.bool,
     /** Information about pending notifications */
     notifications: PropTypes.object,
+    /**
+     * Additional props to be spread to the popovers of the action menu, collapsed mobile menu, notifications menu, product menu,
+     * product switch popover and the profile menu.
+     * */
+    popoverPropsFor: {
+        actionMenu: PropTypes.object,
+        collapsedMobileMenu: PropTypes.object,
+        notifications: PropTypes.object,
+        productMenu: PropTypes.object,
+        productSwitch: PropTypes.object,
+        profileMenu: PropTypes.object
+    },
```

Comment:
How about separating each of them out?
```
Shellbar.PropTypes = {
        actionMenuPopoverProps: PropTypes.object,
        collapsedMobileMenuPopoverProps: PropTypes.object,
        notificationsPopoverProps: PropTypes.object,
        productMenuPopoverProps: PropTypes.object,
        productSwitchPopoverProps: PropTypes.object,
        profileMenuPopoverProps: PropTypes.object
},
```

Reply:
Yeah, I started out with that but thought there are too many popovers. So I think packaging them would be cleaner, no?

Reply:
Do you think we would need to make other changes to the popovers or just placement overrides?

Reply:
We might need to customize some cosmetic props like `noArrow`, `widthSizingType` or add `popperClassName` and other stuff. It would be a little more future proof.